test/fixtures/simple_example/main.tf

```diff
@@ -25,8 +25,36 @@ resource "random_pet" "main" {
 }
 
 module "example" {
-  source = "../../../examples/simple_example"
+  source = "../../../examples/simple"
 
-  project_id  = var.project_id
-  bucket_name = random_pet.main.id
+  project = var.project_id
+  name    = random_pet.main.id
```

Comment:
I see a variable name defined but here it seems to use a random id. Should the var be removed?

Reply:
It seems like the author was passing the name to the example. Hardcoded this in examples.